otherlibs/unix/unixLabels.mli

```diff
@@ -622,6 +622,11 @@ val link : ?follow (* thwart tools/sync_stdlib_docs *) :bool ->
                  unavailable.
    @raise ENOSYS On {e Windows} if [~follow:false] is requested. *)
 
+val realpath : string -> string
+(** [realpath p] is the absolute pathname for [p] obtained by resolving
```

Comment:
Just a nit since the result is not necessarily unique, e.g. due to hardlinks.
```suggestion
(** [realpath p] is an absolute pathname for [p] obtained by resolving
```